Submitted By: Jim Gifford <jim@linuxfromscratch.org>
Date: 2006-10-22
Initial Package Version: 2.5
Origin: Upstream
Upstream Status: From Upstream
Description: Corrects the Minor Version in Glibc

diff -Naur glibc-2.5.orig/include/features.h glibc-2.5/include/features.h
--- glibc-2.5.orig/include/features.h	2006-04-25 13:34:32.000000000 -0700
+++ glibc-2.5/include/features.h	2006-10-23 10:45:04.617073026 -0700
@@ -304,7 +304,7 @@
 /* Major and minor version number of the GNU C library package.  Use
    these macros to test for features in specific releases.  */
 #define	__GLIBC__	2
-#define	__GLIBC_MINOR__	4
+#define	__GLIBC_MINOR__	5
 
 #define __GLIBC_PREREQ(maj, min) \
 	((__GLIBC__ << 16) + __GLIBC_MINOR__ >= ((maj) << 16) + (min))
